frontend/src/data/utils/datasetutils.ts

```diff
@@ -221,5 +221,30 @@ export function maybeApplyRowReorder(rows: Row[], breakdowns: Breakdowns) {
       UNKNOWN_HL
     );
   }
+
+  if (breakdowns.hasOnlyAge()) {
+    finalRows.sort(sortAgeParsedNumerically);
+  }
+
   return finalRows;
 }
+
+export const sortAgeParsedNumerically = (l: any, r: any) => {
+  let lAge = l["age"];
+  let rAge = r["age"]; //Rage hehe
+
+  if (lAge === "All" && rAge === "All") return 0;
+  else if (lAge === "All") return -1;
+  else if (rAge === "All") return 1;
+
+  let leftUnbounded = lAge.indexOf("+") !== -1;
+  let rightUnbounded = rAge.indexOf("+") !== -1;
+
+  if (leftUnbounded && rightUnbounded) return 0;
+  else if (leftUnbounded) return 1;
+  else if (rightUnbounded) return -1;
+
+  let lMin = lAge.split("-")[0];
+  let rMin = rAge.split("-")[1];
```

Comment:
Did you mean to do rAge.split("-")[0] here instead of [1]? I would imagine you want to compare ages A and X when sorting A-B and X-Y.

Reply:
Thinking a bit more about this, I suppose what I am saying vs what you are doing would differ in the case of e.g. 6-11 and 5-7.. mine would order them as 5-7, 6-11, whereas yours would order them as 6-11, 5-7. In the case of non-overlapping buckets this doesn't matter, so I'm fine with leaving as-is, though changing the variable name to rMax instead of rMin would be helpful